vars/runTestFunctionalV1.groovy

```diff
@@ -86,27 +87,32 @@ def call(Map config = [:]) {
                                     rm -rf install/lib/daos/TESTING/ftest/avocado ./*_results.xml
                                     mkdir -p install/lib/daos/TESTING/ftest/avocado/job-results
                                     ftest_arg="%s"
+                                    test_repeat=$(git show -s --format=%%B | sed -ne "/^Test-repeat%s:/s/^.*: *//p")
```

Comment:
You should use [cachedCommitPragma()](https://github.com/daos-stack/pipeline-lib/blob/4dabd3366d2386b52c0580a490e7841070904142/vars/cachedCommitPragma.groovy) for this sort of thing now.  You'd of course have to use it outside of this script and substitute it's value, or I suppose you could use it inline here:
```suggestion
                                    test_repeat=''' + cachedCommmitPragma('…') + '''
```
Probably easier to just use it outside and `format()` the value back in.

Reply:
I was just following what is currently done in the script for `test-tag`: https://github.com/daos-stack/pipeline-lib/blob/4dabd3366d2386b52c0580a490e7841070904142/vars/runTestFunctionalV1.groovy#L46-L49